packages/search-ui-elasticsearch-connector/package.json

```diff
@@ -41,6 +41,6 @@
   "dependencies": {
     "@elastic/elasticsearch": "^8.2.1",
     "@elastic/search-ui": "1.17.1",
-    "@searchkit/sdk": "^3.0.0-canary.58"
+    "@searchkit/sdk": "^3.0.0"
```

Comment:
Is there a functional consequence of this? If so, should we create a separate PR for it?